.gitignore

```diff
@@ -1,4 +1,5 @@
 .env
 node_modules/
 build
-.eslintcache
\ No newline at end of file
+.eslintcache
+.ds_store
```

Comment:
annoying mac thing

Reply:
Could you possibly split this out into two commits for the sake of a more descriptive commit history? One `chore` commit for this and one `feat(showcase)` commit for the success message?

Reply:
@evanronnei sure thing, I'll split those out